### DISCUSSION TITLE
Private Internet Access and dedicated IP

### DISCUSSION THREAD

**Discussion by 60plus** [qdm12/gluetun#1002] [Q&A]
Hi.
I have question about Gluetun PIA and dedicated IP.
How to configure it in docker i did not find any env variebles for dedicated IP on github or Gluetun wiki.

**Comment by qdm12** [comment]
What do you mean by 'dedicated IP'?
Can you link up to something about this on PIA's website?

**Reply by 60plus** [reply]
https://www.privateinternetaccess.com/vpn-features/dedicated-ip-vpn

**Reply by qdm12** [reply]
I created #1003 for it, please subscribe to that issue to be alerted when this is supported. It might take a few months I'm quite short-handed/short-staffed (me and doing this on evenings and weekends) currently 😄

**Reply by 60plus** [reply]
it works in such a way that PIA gives ip permanently for a fee for a specified period of time

**Reply by 60plus** [reply]
![obraz](https://user-images.githubusercontent.com/73450566/170520153-0961f670-01d3-4181-8f41-50522d006d23.png)

**Comment by jordanlambrecht** [comment]
Did you ever figure this out?
